Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -9,7 +9,7 @@ $ErrorActionPreference = 'Stop'
 
 $toolVersion = @{
     dbDeploy = "2.0.0-b10001"
-    codeGen = "4.0.0-b10044"
+    codeGen = "4.0.0-b10080"
```

Comment:
Did we miss a code gen utility package update at some point previously?  I'm not seeing any changes from your PR to the codeGen, so wouldn't expect us to package bump this.

Reply:
Reverted it back to b10044